docker/Dockerfile

```diff
@@ -46,12 +39,8 @@ RUN curl https://phar.phpunit.de/phpunit-5.7.5.phar -L -o phpunit.phar \
 	&& chmod +x phpunit.phar \
```

Comment:
We'll have to install different versions of `phpunit` depending on php-version:

![image](https://user-images.githubusercontent.com/87168/38606272-3bceb176-3d7e-11e8-8fc9-75bb43f1b46b.png)

Specifically  `phpunit-5.7.5` won't work with `php-7.2`.

Not sure if our tests support phpunit6 yet, though. I remember fiddling with them a bit to get them work some months ago.

via https://phpunit.de